docs/testing/automated_benchmark_test.md

```diff
@@ -17,6 +17,8 @@ apt install python3-paramiko
 ## Extra configuration on hypervisors running Gardenlinux
 On hypervisors running gardenlinux, it is also necessary to open ports to allow the DHCP service to provide IP addresses to VMs to be able for access. For example, the most convenient way is to  change the default input filter policy to 'accept' by importing the following nft table rules.
 
+The provided script, `hack/connectivity_test/prepare_hypervisor.sh`, can perform the following actions at one time.
+
 ```
 command: sudo nft -f filter_table.nft
```

Comment:
I saw that you removed this iptables rule from the prepare script. Would it sill make sense to keep it here ?